README.md

```diff
@@ -203,7 +203,35 @@ COPY bin/my-wag-service /usr/bin/my-wag-service
 
   * If you don't have a context to pass to a Wag function you have two options
     * context.Background() - use this when this is the creator of the request chain, like a test or a top-level service.
-    * context.TODO() - use this when you haven't been passed a context from a caller yet, but you expect the caller to send you one at some point.  
+    * context.TODO() - use this when you haven't been passed a context from a caller yet, but you expect the caller to send you one at some point.
+
+
+### DynamoDB Codegen
+
+  * Wag can auto-generate server code to save models to DynamoDB if you specify the `x-db` extension on a schema:
+  ```yaml
+definitions:
+  Thing:
+    x-db:
```

Comment:
these docs appear inconsistent with usage in `workflow-manager` https://github.com/Clever/workflow-manager/pull/133/files#diff-65058f891490afbb80c8704444e0af9bR392